modules/acknowledgements/jsx/acknowledgementsIndex.js

```diff
@@ -274,7 +276,7 @@ class AcknowledgementsIndex extends Component {
             name='addStartDate'
             label='Start date'
             value={this.state.formData.addStartDate}
-            maxYear={this.state.data.maxYear}
+            maxYear={this.state.formData.addEndDate || this.state.data.maxYear}
```

Comment:
The maxYear comes from configuration settings.  This fix can set the year greater than maxYear.
the same issue for minYear.

Reply:
i'm not sure i understand what you mean. what this fix does is set the max year to be whatever you input into the 'end date' field (so that start date can't be after end date) if it exists. if you haven't put anything into the 'end date' field yet, the max year would be the config max year. does this not make sense for this form?

Reply:
@kongtiaowang I think you're thinking about the new_profile module and this fix by @zaliqarosli is just to make sure the end_date is not before the start_date of the acknowledgements module.